backend/README.md

```diff
@@ -1,350 +1,358 @@
+
+> src/main/resources/application[-profile].yml 
+
+You will have to configure the 'datasource' property for each application.
+
+There are some specific settings in the 'application' section of the configuration that might need customization:
+
+> crawler-service 
+
+```
+application.repoistory.github.token
+```
+enter your github token to bypass the 60 calls/hour limit
+
+```
+application.repository.origin.url
+application.repository.origin.branch
+```
+repository where the master list for webcomponents can be found. must use https and not ssh
+
+```
+application.workspace.path
+```
+path to the local file system where the crawler will store the "dist" files.
+
+> delivery-service 
+
+```
+application.workspace.path
+```
+path to the local file system where the crawler has stored the "dist" files
+
+> data-service 
+
+```
+application.workspace.path
+```
+path to the local file system where the crawler has stored the "dist" files
+
+```
+application.deliveryBaseUrl
+```
+url where the frontend can reach the delivery service to load dist files
+
+## Deployment
+
+The project requires a Tomcat server as well as a PostgreSQL 9.6+ database server.
+
+Build the project
+
+```bash
+mvn clean package
+```
+
+If you want to indivually build each module, you have to make sure that the common module is built first. eg:
+```bash
+cd common
+mvn clean install
+cd data-service
+mvn clean package
+```
+
+There are three services that need to be deployed.
+
+You will find the .war files to deploy in the 'target' folder of each service.
+
+For an initial setup deploy 'data-service' first to have the database populated with tables.
+
+## API & swagger
+
+> data-service
+
+data-service will expose the main api for webcomponents. it will also expose a swagger-ui (/swagger-ui.html)
+
+> delivery-service
+
+delivery-service will expose api for retrieving the dist files needed to embed webcomponents. it will also expose a swagger-ui (/swagger-ui.html)
+
+## General Usage
+
+### Adding a webcomponent
+
+Adding a new webcomponent requires an entry in the main json file. This file can be found in the repository configured as "application.repository.origin.url".
+
+> origins.json
+
+1) Use a UUID-generator to create a new id
+2) Create a new entry in the origins file referencing the repository of the webcomponent
+3) Add at least one version-tag to the reposotory of the webcomponent. Version tags must look like this: v1.0
+
+The repository of the new webcomponent must have at least one version tag and contain a wcs-manifest.json file!
+
+### wcs-manifest.json
+
+To register with the webcomponents hub a wcs-manifest.json file must be in your repository's root directory. A manifest file should look like this:
+
+```
+{
+  "title": "Generic Map",
+  "description": "Generic Map to access the Open Data Hub Mobility API v2 (with Ninja)",
+  "descriptionAbstract": "Generic Map to access the Open Data Hub Mobility",
+  "license": "GPL-3.0",
+  "copyrightHolder": [

```

Comment:
@uhufa Isn't it copyrightHolders (plural)?

Reply:
sorry, my bad. it is "copyrightHolders"